File: scratch/search_rp.py
import os

workspace_dir = r"c:\Users\mohgh\analisis-data-py"
files_to_check = ["index.html", "script.js"]

for filename in files_to_check:
    filepath = os.path.join(workspace_dir, filename)
    if os.path.exists(filepath):
        print(f"Checking {filename}...")
        with open(filepath, "r", encoding="utf-8") as f:
            lines = f.readlines()
            for idx, line in enumerate(lines):
                if "Rp" in line or "rupiah" in line.lower():
                    print(f"Line {idx+1}: {line.strip()}")

File: scratch/search_rp.js
const fs = require('fs');
const path = require('path');

const workspaceDir = 'c:\\Users\\mohgh\\analisis-data-py';
const filesToCheck = ['index.html', 'script.js'];

filesToCheck.forEach(filename => {
    const filepath = path.join(workspaceDir, filename);
    if (fs.existsSync(filepath)) {
        console.log(`Checking ${filename}...`);
        const content = fs.readFileSync(filepath, 'utf8');
        const lines = content.split('\n');
        lines.forEach((line, idx) => {
            if (line.includes('Rp') || line.toLowerCase().includes('rupiah')) {
                console.log(`Line ${idx + 1}: ${line.trim()}`);
            }
        });
    }
});
